util/pkgconfig/seqan.pc.in

```diff
@@ -6,5 +6,6 @@ Name: @CMAKE_PROJECT_NAME@
 Description: C++ library for biological sequence analysis
 URL: http://www.seqan.de
 Version: @SEQAN_VERSION_STRING@
-Requires: zlib
-Cflags: -I${includedir}
+Requires: @SEQAN_PKGCFG_DEPS@
+Cflags: -I${includedir}  @SEQAN_PKGCFG_CFLAGS@ -DSEQAN_HAS_ZLIB=1 -DSEQAN_HAS_BZIP2=1 -DSEQAN_HAS_OPENMP=1
```

Comment:
We don't need ` -DSEQAN_HAS_OPENMP=1`, anymore, the code only relies on `_OPENMP`.

But to be able to use this we would instead need `-fopenmp`. Do you think it makes sense to hardcode this? It means it won't work on clang < 3.8 .